### PR TITLE
List invalid xml

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-blob/swagger/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ autorest --java --use:@autorest/java@4.0.x
 
 ### Code generation settings
 ``` yaml
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/storage-dataplane-preview/specification/storage/data-plane/Microsoft.BlobStorage/preview/2020-12-06/blob.json
+input-file: https://raw.githubusercontent.com/seanmcc-msft/azure-rest-api-specs/feature/storage/listBlobsInvalidXmlCharacters/specification/storage/data-plane/Microsoft.BlobStorage/preview/2021-02-12/blob.json
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-blob/swagger/README.md:19]: Remember to change this back.  IIRC, STG 80 swagger is now in the main branch - https://github.com/Azure/azure-rest-api-specs/tree/main/specification/storage/data-plane

File: sdk/storage/azure-storage-file-datalake/swagger/src/main/java/com/azure/storage/file/datalake/customization/DataLakeStorageCustomization.java [hunk 2]
```diff
@@ -8,13 +8,14 @@
...
 import com.azure.autorest.customization.PackageCustomization;
+import org.slf4j.Logger;
 
 /**
  * Customization class for File DataLake Storage.
  */
 public class DataLakeStorageCustomization extends Customization {
     @Override
-    public void customize(LibraryCustomization customization) {
+    public void customize(LibraryCustomization customization, Logger logger) {
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-file-datalake/swagger/src/main/java/com/azure/storage/file/datalake/customization/DataLakeStorageCustomization.java:18]: What is this?

Comment (rickle-msft) [reply sdk/storage/azure-storage-file-datalake/swagger/src/main/java/com/azure/storage/file/datalake/customization/DataLakeStorageCustomization.java:18]: Autorest had a breaking change in their interface. This was a necessary change to generate new code

**Review by seanmcc-msft — APPROVED** [review]

**Pull request merged**